### PR TITLE
luafilesystem, bump version, cleanup

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: dev-lua/luafilesystem/luafilesystem-1.7.0.2.recipe [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
+directory structure and file attributes."
 HOMEPAGE="http://keplerproject.github.io/luafilesystem/"
-COPYRIGHT="2004-2012 Kepler Project"
+COPYRIGHT="2003 - 2017 Kepler Project"
 LICENSE="MIT"
 REVISION="1"
-SOURCE_URI="https://github.com/keplerproject/luafilesystem/archive/v_1_6_3.tar.gz"
-CHECKSUM_SHA256="5525d2b8ec7774865629a6a29c2f94cb0f7e6787987bf54cd37e011bfb642068"
-SOURCE_DIR="luafilesystem-v_1_6_3"
+SOURCE_URI="https://github.com/keplerproject/luafilesystem/archive/v1_7_0_2.tar.gz"
```
Comment (fbrosson) [inline dev-lua/luafilesystem/luafilesystem-1.7.0.2.recipe]:
```diff
-SOURCE_URI="https://github.com/keplerproject/luafilesystem/archive/v1_7_0_2.tar.gz"
+SOURCE_URI="https://github.com/keplerproject/luafilesystem/archive/v${portVersion//\./_}.tar.gz"
```

File: dev-lua/luafilesystem/luafilesystem-1.7.0.2.recipe [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
+CHECKSUM_SHA256="23b4883aeb4fb90b2d0f338659f33a631f9df7a7e67c54115775a77d4ac3cc59"
+SOURCE_DIR="luafilesystem-1_7_0_2"
```
Comment (fbrosson) [inline dev-lua/luafilesystem/luafilesystem-1.7.0.2.recipe]:
```diff
-SOURCE_DIR="luafilesystem-1_7_0_2"
+SOURCE_DIR="luafilesystem-${portVersion//\./_}"
```

File: dev-lua/luafilesystem/luafilesystem-1.7.0.2.recipe [hunk 3]
```diff
@@ -1,16 +1,15 @@
...
-related to file systems offered by the standard Lua distribution.
-LuaFileSystem offers a portable way to access the underlying directory \
-structure and file attributes."
+DESCRIPTION="LuaFileSystem is a Lua library developed to complement the set \
+of functions related to file systems offered by the standard Lua \
+distribution. LuaFileSystem offers a portable way to access the underlying \
+directory structure and file attributes."
 HOMEPAGE="http://keplerproject.github.io/luafilesystem/"
-COPYRIGHT="2004-2012 Kepler Project"
+COPYRIGHT="2003 - 2017 Kepler Project"
```
Comment (korli) [inline dev-lua/luafilesystem/luafilesystem-1.7.0.2.recipe]: Why add spacing?

File: dev-lua/luafilesystem/luafilesystem-1.7.0.2.recipe [hunk 4]
```diff
@@ -46,6 +45,14 @@ INSTALL()
...
 	LUAVERSION=$(lua -v | sed 's/Lua //' | head -c3)
 	make LUA_LIBDIR="$libDir/lua/$LUAVERSION" install
+
+#	copy the doc files
+#	mkdir -p $docDir
+#	cp -r doc/* $docDir
+
+#	copy the header file
+#	mkdir -p $includeDir
+#	cp src/lfs.h $includeDir
```
Comment (korli) [inline dev-lua/luafilesystem/luafilesystem-1.7.0.2.recipe]: Is it intended to let these lines commented?

Comment (Begasus) [reply dev-lua/luafilesystem/luafilesystem-1.7.0.2.recipe]: Aside from wordgrinder I haven't found any recipe looking for the header file, so commenting them out for now

File: dev-lua/luafilesystem/luafilesystem-1.7.0.2.recipe (hunk 3 above)
Comment (Begasus) [reply dev-lua/luafilesystem/luafilesystem-1.7.0.2.recipe]: My bad, cp'd from source :)

File: dev-lua/luafilesystem/luafilesystem-1.7.0.2.recipe [hunk 5]
```diff
@@ -46,6 +45,14 @@ INSTALL()
...
+#	copy the doc files
+#	mkdir -p $docDir
+#	cp -r doc/* $docDir
+
+#	copy the header file
+#	mkdir -p mkdir -p $includeDir
+#	cp src/lfs.h $includeDir
 }
 
 TEST()
```
Comment (Begasus) [inline dev-lua/luafilesystem/luafilesystem-1.7.0.2.recipe]: Test still fails ...

File: dev-lua/luafilesystem/luafilesystem-1.7.0.2.recipe [hunk 6]
```diff
@@ -46,6 +45,14 @@ INSTALL()
...
 	LUAVERSION=$(lua -v | sed 's/Lua //' | head -c3)
 	make LUA_LIBDIR="$libDir/lua/$LUAVERSION" install
+
+#	copy the doc files
+#	mkdir -p $docDir
+#	cp -r doc/* $docDir
+
+#	copy the header file
+#	mkdir -p mkdir -p $includeDir
+#	cp src/lfs.h $includeDir
```
Comment (Begasus) [inline dev-lua/luafilesystem/luafilesystem-1.7.0.2.recipe]: Not sure if these are needed, no package has been depending on them ...

Comment (pulkomandy) [reply dev-lua/luafilesystem/luafilesystem-1.7.0.2.recipe]: If "make install" does not install them, it means upstream does not think they are needed. So, don't copy them.

**Comment by scottmc** [comment]
@Begasus can you look into wrapping this one up?

**Pull request merged**